src/stream/src/task/stream_manager.rs

```diff
@@ -707,6 +727,8 @@ impl LocalStreamManagerCore {
     fn drop_all_actors(&mut self) {
         for (actor_id, handle) in self.handles.drain() {
             self.context.retain(|&(up_id, _)| up_id != actor_id);
+            self.actor_monitor_tasks.get(&actor_id).unwrap().abort();
+            self.actor_monitor_tasks.remove(&actor_id);
```

Comment:
can just change the above two lines to `remove.unwrap().abort()`

Reply:
Sure